default/mail_tt2/report.tt2

```diff
@@ -384,6 +384,9 @@
         [%|loc(report_param.name,report_param.listname)%]Including subscribers from data source "%1" into list %2 failed.[%END%]
     [%~ END %]
 
+[%~ ELSIF report_entry == 'include_incomplete' ~%]
+   [%|loc(report_param.result.added,report_param.result.held,report_param.result.updated)%]Inclusion partially or entirely failed (%1 added, %2 held, %3 updated).[%END%]
+
```

Comment:
I would prefer to break this in two cases (partially / entirely).

We can determine whether a inclusion was partially or entirely failed, so I don't see a point in mixing them together. I also assume that in most cases we have a single datasource, so no need to confuse these users with partially failures.

Reply:
Partial and entire cases were separated. See ee42f48